packages/components/simple-table/src/cell.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+import { AngleDownIcon, AngleUpIcon } from '@commercetools-uikit/icons';
+import AccessibleButton from '@commercetools-uikit/accessible-button';
+import omit from 'lodash/omit';
+import { getCellPadding, BaseCell, BaseHeaderCell } from './cell.styles';
+
+const SortableHeaderCell = props => {
+  const isActive = props.sortedBy === props.columnKey;
+
+  const Icon =
+    isActive && props.sortDirection === 'desc' ? AngleDownIcon : AngleUpIcon;
+
+  const IconComponent = styled(Icon)`
+    visibility: ${isActive ? 'visible' : 'hidden'};
+  `;
+  const ButtonContainer = styled(AccessibleButton)`
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex: 1;
+
+    padding: ${getCellPadding(props.isCondensed)};
+
+    & ${IconComponent} {
+      margin-left: ${vars.spacingS};
+    }
+
+    &:hover,
+    &:focus {
+      cursor: pointer;
+
+      ${IconComponent} {
+        visibility: visible;
+        * {
+          fill: ${vars.colorNeutral};
+        }
+      }
+    }
+  `;
+
+  return (
+    <BaseHeaderCell
+      {...omit(props, ['onSortChange'])}
```

Comment:
Nit: maybe `const baseHeaderCellProps` ontop could ease reading this.

Reply:
Yes, I'm not too happy with this and I'm actually going to refactor the `BaseHeaderCell`, and also make them fully accessible when they're clickable.

Might still use your tip though!